Obfuscator.ge

Resuscitating opened the pull request.

Commit:
Merge pull request #3 from thatbirdguythatuknownot/main

update to latest version

Commit:
Merge branch 'thatbirdguythatuknownot:main' into main

Commit:
add Obfuscator.ge

Obfuscator.ge, as well as some minor housekeeping+bugfixing

ghost commented:
there's still a lot of things it doesn't support, but it supports `str`s, `dict`s, `int`s, `tuple`s, and any instance of `type`
will work on improving it and adding more functionality over the next few days

thatbirdguythatuknownot commented:
Shouldn't `ge_cache` be reset in `Obfuscator.c()`?
Also what's with the removed newlines?

Commit:
fix spacing in rules

for some reason it deleted the newlines in the docstring so I'm putting them back

thatbirdguythatuknownot commented:
I guess I'll merge this, manually integrate it into the local file so I can do some fixes, and update the online copy. Thanks!

ghost commented:
> Shouldn't ge_cache be reset in Obfuscator.c()?
> Also what's with the removed newlines?

Yeah, I'm not sure what's going on with the NLs, they got removed for some reason. They've been added back.
As for the cache, you're right, I just forgot to do that.
